stable/stable_rec/Readme.md

```diff
@@ -0,0 +1,65 @@
+## Test suite for Kubernetes
```

Comment:
stable_rec === ? is this a copy of the original ... if yes, can you drop this folder ?

Reply:
stable_rec and stable_gen serve to split the parent directory into two sub-directories each containing the necessary code for one of the connection components.